### PR TITLE
Trend Micro Vision One 1.1

### PR DESCRIPTION
**Pull request by mikedgibson**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added two new integration commands allowing alert notes to be added and alert statuses to be changed.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [x] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/TrendMicroVisionOne/ReleaseNotes/1_1_0.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+- Added a new command 'trendmicro-visionone-update-status' to modify the status of a Trend Micro Vision One workbench alert
```
Comment (kgal-pan) [inline Packs/TrendMicroVisionOne/ReleaseNotes/1_1_0.md:3]:
```suggestion
- Added a new command ***trendmicro-visionone-update-status*** to modify the status of a Trend Micro Vision One workbench alert
- Added a new command ***trendmicro-visionone-add-note*** to add a note to Trend Micro Vision One workbench alert
```

File: Packs/TrendMicroVisionOne/ReleaseNotes/1_1_0.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+- Added a new command 'trendmicro-visionone-add-note' to add a note to Trend Micro Vision One workbench alert
```
Comment (kgal-pan) [inline Packs/TrendMicroVisionOne/ReleaseNotes/1_1_0.md:4]:
```suggestion
- Added a new command ***trendmicro-visionone-add-note*** to add a note to Trend Micro Vision One workbench alert
```

File: Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOne/TrendMicroVisionOne.py [hunk 3]
```diff
@@ -187,7 +203,8 @@ def http_request(self, method: str, url_suffix: str, json_data=None, params=None
         """
         header = {
             "Authorization": "Bearer {token}".format(token=self.api_key),
-            "Content-Type": f'{CONTENT_TYPE_JSON}{";charset=utf-8"}'
+            "Content-Type": f'{CONTENT_TYPE_JSON}{";charset=utf-8"}',
+            "User-Agent": f'{USER_AGENT}'
```
Comment (yaakovi) [inline Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOne/TrendMicroVisionOne.py:207]:
```suggestion
            "Content-Type": f'{CONTENT_TYPE_JSON};charset=utf-8',
            "User-Agent": USER_AGENT,
```

**Review by yaakovi — COMMENTED** [review]
looks good.
please add a unit test for the new commands.

File: Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOne/TrendMicroVisionOne.py (hunk 3 above)
Comment (mikedgibson) [reply Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOne/TrendMicroVisionOne.py]: Thanks for the suggestion. Change incorporated.

**Comment by kgal-pan** [comment]
@mikedgibson - Please run:

```bash
./Utils/git_pull_master_into_fork.sh
```

**Comment by mikedgibson** [comment]
Hi Kobbi. I did do that as the last step before I requested a review and it looked like the build was successful after that. Do I need to run it again?

**Comment by kgal-pan** [comment]
Thanks Mike, no need to rerun.

File: Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOne/TrendMicroVisionOne_test.py [hunk 4]
```diff
@@ -713,3 +715,62 @@ def test_get_endpoint_information(mocker):
...
+        update_status_mock_response)
+    client = Client("https://api.xdr.trendmicro.com", api_key)
+    args = {
+        "workbench_id": "WB-20837-20220418-00000",
+        "status": "in_progress"
+    }
+    result = update_status(client, args)
+    assert result.outputs["response_msg"] == "Alert status changed successfully."
+    assert isinstance(result.outputs["Workbench_Id"], str)
+    assert isinstance(result.outputs["response_code"], int)
```
Comment (yaakovi) [inline Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOne/TrendMicroVisionOne_test.py:776]:
```suggestion
    assert isinstance(result.outputs["response_code"], int)

```

File: Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOne/TrendMicroVisionOne_test.py [hunk 5]
```diff
@@ -13,7 +13,9 @@
     download_information_collected_file,
     submit_file_to_sandbox,
     get_task_status,
-    get_endpoint_info
+    get_endpoint_info,
+    add_note,
+    update_status
```
Comment (yaakovi) [inline Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOne/TrendMicroVisionOne_test.py:18]:
```suggestion
    update_status,
```

**Review by yaakovi — APPROVED** [review]

**Pull request merged**